distribution/ddf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -21,11 +21,9 @@
 # Comma separated list of features repositories to register by default
 #
 featuresRepositories= \
```

Comment:
❓ `distribution/kernel/src/main/filtered-resources/etc/org.apache.karaf.features.cfg` has `mvn:${project.groupId}/${project.artifactId}/${project.version}/xml/features` as a feature repository. Now that it doesn't look like kernel has a feature file anymore, could we remove that feature repo as well?